### PR TITLE
Refine explanation of meeting regenerate after expiry requirement

### PR DESCRIPTION
**Pull request by deads2k**
/assign @vrutkovs @sanchezl

### CODE REVIEW

File: tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md [hunk 1]
```diff
@@ -39,17 +40,23 @@
...
 the cluster will automatically create new cert/key pairs or update CA bundles as required without human
 intervention.
-To assert that a particular cert/key pair or CA bundle can do this, add the "certificates.openshift.io/auto-regenerate-after-offline-expiry" annotation to the secret or configmap and 
-setting the value of the annotation a github link to the PR adding the annotation.
-This assertion also means that you have
+
+To assert that a particular cert/key pair or CA bundle can do this, add the annotation to the secret or configmap.
+```yaml
+  annotations:
+    certificates.openshift.io/auto-regenerate-after-offline-expiry: https//github.com/link/to/pr/adding/annotation, "quote escaped formatted name of e2e test that ensures the PKI artifact functions properly"
```
Comment (vrutkovs) [inline tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md:52]: I think just the test name should be sufficient, we can find out which PR added it with `git blame` (also across releases)

File: tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md [hunk 2]
```diff
@@ -39,17 +40,23 @@
...
+```
+
+This assertion means that you have
 1. Manually tested that this works or seen someone else manually test that this works.  AND
-2. Written an automated e2e job that your team has an alert for and is a blocking GA criteria, and/or
+2. Written an automated e2e test to ensure this PKI artifact is function that is a blocking GA criteria, and/or
       QE has required test every release that ensures the functionality works every release.
-Links should be provided in the PR adding the annotation.
+If you have not done this, you should not merge the annotation.
```
Comment (vrutkovs) [inline tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md:59]:
```suggestion
If you have not done this, you should not merge the PR setting this annotation.
```

File: tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md (hunk 1 above)
Comment (deads2k) [reply tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md:52]: > I think just the test name should be sufficient, we can find out which PR added it with `git blame` (also across releases)

Based on past experience in kube with api approvals, there's nothing quite like the link embedded.

**Comment by williamcaban** [comment]
The clarifications included in this PR for the generated document about how to meet the requirement provide the information PMs and Eng teams need to better understand what they need to do.

File: tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md (hunk 2 above)
Comment (deads2k) [reply tls/autoregenerate-after-expiry/autoregenerate-after-expiry.md:59]: @vrutkovs based on William's comment, I'm going to merge this to help us communicate where we're at with a perma-link.

I'm completely fine adding this extra verbiage, but I need to merge with green verifies.

**Comment by deads2k** [comment]
merging to allow linking for email/jira/etc.

**Pull request merged**